widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.kt

```diff
@@ -419,6 +421,10 @@ object GliaWidgets {
         }
     }
 
+    fun getPushNotifications(): PushNotifications {
```

Comment:
Will you add `@JvmStatic` ? 🙂

Reply:
I just saw you added it in [another PR](https://github.com/salemove/android-sdk-widgets/pull/1278/files).